linera-core/src/updater.rs

```diff
@@ -482,7 +506,7 @@ where
         &mut self,
         action: CommunicateAction,
     ) -> Result<LiteVote, ChainClientError> {
-        let (target_block_height, chain_id) = match &action {
+        let (_target_block_height, chain_id) = match &action {
```

Comment:
Let's remove that from all the cases then; this line should just be: `let chain_id = action.chain_id()`.